packages/wasm/package.json

```diff
@@ -30,12 +30,154 @@
-    "test": "pw-test --mode node && pw-test"
+    "build": "export PROFILE=dev && export TARGET_DIR=debug && pnpm run build:all",
+    "release": "export PROFILE=release && export TARGET_DIR=release && pnpm run build:all",
+    "build:all": "wireit",
+    "clean": "wireit",
+    "test": "wireit",
+    "test:node": "wireit"
+  },
+  "wireit": {
+    "compile": {
+      "command": "cargo build --target wasm32-unknown-unknown --profile $PROFILE",
+      "env": {
+        "PROFILE": {
+          "external": true
+        }
+      }
+    },
+    "opt": {
+      "command": "wasm-opt -O4 target/wasm32-unknown-unknown/$TARGET_DIR/wasm.wasm -o target/wasm32-unknown-unknown/$TARGET_DIR/wasm.wasm",
+      "env": {
+        "TARGET_DIR": {
+          "external": true
+        }
+      },
+      "dependencies": [
+        "compile"
+      ]
+    },
+    "bindgen:bundler": {
+      "command": "wasm-bindgen --weak-refs --target bundler --out-dir dist/bundler target/wasm32-unknown-unknown/$TARGET_DIR/wasm.wasm",
+      "env": {
+        "TARGET_DIR": {
+          "external": true
+        }
+      },
+      "dependencies": [
+        "opt"
+      ],
+      "output": [
+        "dist/bundler"
+      ]
+    },
+    "bindgen:nodejs": {
+      "command": "wasm-bindgen --weak-refs --target nodejs --out-dir dist/nodejs target/wasm32-unknown-unknown/$TARGET_DIR/wasm.wasm && move-file dist/nodejs/wasm.js dist/nodejs/wasm.cjs",
+      "env": {
+        "TARGET_DIR": {
+          "external": true
+        }
+      },
+      "dependencies": [
+        "opt"
+      ],
+      "output": [
+        "dist/nodejs"
+      ]
+    },
+    "bindgen:web": {
+      "command": "wasm-bindgen --weak-refs --target web --out-dir dist/web target/wasm32-unknown-unknown/$TARGET_DIR/wasm.wasm && cpy --flat src/workerd.js dist/web",
+      "env": {
+        "TARGET_DIR": {
+          "external": true
+        }
+      },
+      "dependencies": [
+        "opt"
+      ],
+      "output": [
+        "dist/web"
+      ]
+    },
+    "build:all": {
+      "dependencies": [
+        "bindgen:bundler",
+        "bindgen:nodejs",
+        "bindgen:web"
+      ]
+    },
+    "clean": {
+      "command": "rimraf dist"
+    },
+    "test:prepare": {
+      "command": "npx playwright install && cross-env mkdir test/report",
+      "output": [
+        "test/report"
+      ]
+    },
+    "test:chromium": {
+      "command": "pw-test test/wasm.test.js -r mocha --reporter json --cov > test/report/chromium.json",
+      "dependencies": [
+        "build",
+        "test:prepare"
+      ],
+      "output": [
+        "test/report/chromium.json"
+      ]
+    },
+    "test:firefox": {
+      "command": "pw-test test/wasm.test.js -r mocha --reporter json --browser firefox > test/report/firefox.json",
```

Comment:
Using the `-r mocha` argument explicitly because `playwright-test` emits an auto-detection message which ends up in the test result files. 

It might be worth adding a `quiet` mode to `playwright-test` when you only want the reporter output. If this seems like a good feature, happy to open an issue over there.

Reply:
please do

Reply:
Done: https://github.com/hugomrdias/playwright-test/issues/571